pkgs/tools/networking/wireguard-go/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, buildGoPackage, fetchurl }:
+
+buildGoPackage rec {
+  name = "wireguard-go-${version}";
+  version = "0.0.20180514";
+
+  goPackagePath = "wireguard-go";
+
+  src = fetchurl {
```

Comment:
As far as I know, cgit snapshots are not deterministic, so this should be `fetchzip` (that checksums archive contents instead of archive itself).

Reply:
🤷‍♂️. I copy-pasted it from the WireGuard expression. Do you think I should fix it in both PRs?

Reply:
Yes. See: https://github.com/nlopes/cgit/blob/9b7473549bf7eda7806c83e3a4a07b30dc36bcaf/ui-snapshot.c

Reply:
✅